### ISSUE TITLE
Scrolling broken in iOS 13.1.2 Safari

### ISSUE THREAD

**Issue by mtbradle-ge** [vaadin/vaadin-grid#1670]
### Description
On iPad with iOS version 13.1.2, scrolling is broken.

### Expected outcome
If there are more elements than the height of the grid, a scrollbar should be present / the user should be able to scroll through the grid (vertically).

### Actual outcome
The grid is not scrollable at all.

### Live Demo
vaadin-grid's own live demo suffices to show the issue:
https://vaadin.com/components/vaadin-grid/html-examples

The table in the example is not scrollable. This seems to be happening on iPad Safari after upgrading to version 13.1.2 (have not checked other browsers on iPad).

### Steps to reproduce
1. Put a `vaadin-grid` element in the page with enough items where scrolling should happen.
2. Open the page in Safari on an iPad that has been upgraded to version 13.1.2

### Browsers Affected
- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE 11
- [x] iOS Safari
- [ ] Android Chrome

### More info
I dug into this some and it looks like one issue is how the scroll bar width is computed in the ScrollMixin. Inside of ```vaadin-grid-scroll-mixin.html```:

```
_scrollbarWidth: {
          type: Number,
          value: function() {
            // Create the measurement node
            var scrollDiv = document.createElement('div');
            scrollDiv.style.width = '100px';
            scrollDiv.style.height = '100px';
            scrollDiv.style.overflow = 'scroll';
            scrollDiv.style.position = 'absolute';
            scrollDiv.style.top = '-9999px';
            document.body.appendChild(scrollDiv);
            // Get the scrollbar width
            var scrollbarWidth = scrollDiv.offsetWidth - scrollDiv.clientWidth;
            // Delete the DIV
            document.body.removeChild(scrollDiv);
            return scrollbarWidth;
          }
        },
```

It looks like in iOS Safari 13.1.2 the offsetWidth and the clientWidth of elements is coming back the same. And therefore the _scrollbarWidth gets set to zero, which in turn disables the scrollbar (looks like there is a no-scrollbars attribute that depends on this value).

Right now our dirty workaround is manually updating the _scrollbarWidth property to a sensible value.

**Comment by tomivirkki** [comment]
Thanks for reporting. Looks like [detection for ios](https://github.com/vaadin/vaadin-grid/blob/master/src/vaadin-grid.html#L368) is no longer valid for iOS13 iPad. Fix in #1671 A workaround for this meanwhile:
```js
grid._ios = (/iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream)
              || (navigator.platform === 'MacIntel' && navigator.maxTouchPoints > 1);
```

**Comment by lucastanzani** [comment]
I should add that on iPhone, only with iOS 13,  when scrolling, it resets back to top if the scrolling is repeated. If the inertial movement is left on its own, it correctly stops and doesn't scroll to top... as soon as you don't scroll again.
